bowtie/_cli.py

```diff
@@ -1713,6 +1918,11 @@ async def _run(
             STDERR.print("[bold red]No test cases ran.[/]")
         elif count > 1:  # XXX: Ugh, this should be removed when Reporter dies
             STDERR.print(f"Ran [green]{count}[/] test cases.")
+
+        if time_measurement_path:
+            with time_measurement_path.open("a") as file:
```

Comment:
Use `Path.write_text`.

Reply:
Oh. You're appending here?

Reply:
Yeah because the same benchmark would run the same test multiple times to avg out, we would need all that value linked to one benchmark only. So one file per benchmark test.